docs/guides/beginnerCLI.rst

```diff
@@ -0,0 +1,276 @@
+        :width: 600
+
+* Once you have downloaded and launched the installer, select the *Install Now* option, and take care to remember the path its installing to. 
+
+    .. image:: ../_static/Images/pythonDownload.png
+        :align: 600
+    
+* Once the installation is complete, press the Windows key and type *Environment Variables* - select the best match that comes up: Edit the System Envrionment Variables.
+
+    .. image:: ../_static/Images/env.png
+        :width: 400
+
+* Select *Environment Variables* at the bottom of the window.
+
+        .. image:: ../_static/Images/environmentV.png
+            :width: 600
+
+* This is where the path from earlier comes in. Under the 'User variables' section, double click the ``path`` variable. Then select *New*. Enter the path where you installed Python. Add ``\Scripts\`` at the end. After select OK in both the edit window and the Environment Variables window. 
+    
+    **IMPORTANT** - If you do not select OK in both windows, it will not save the update and you will have to repeate this step.
+
+        .. image:: ../_static/Images/path.png
+            :width: 800
+    
+Installing beets
+^^^^^^^^^^^^^^^^
+2.  Press the Windows key and type 'cmd', and then press enter on the Command Prompt. This will open your Command Line Interface. Type ``cd [folder name]`` that you wish to install beets into. Note that you may have to change directories a few times to get to the desired one ex. ``cd users\name\music_library``. For this example, I am saving it to my user so I do not need to change directories. 
+ 
+* Now install beets by running: ``pip install beets``. A successful install will collect and download the included libraries.
+
+    .. image:: ../_static/Images/install.png
+        :width: 800
+
+* You're all set! Type ``beet`` at the command prompt to make sure everything's in order. Doing so will bring up a list of helpful commands, as well as the format needed to use them.
+
+    .. image:: ../_static/Images/command.png
+        :width: 600
+
+**Optional** - You may also want to install a context menu item for importing files into beets. Download the `beets.reg`_ file and open it in a text file to make sure the paths to Python match your system. Then double-click the file add the necessary keys to your registry. You can then right-click a directory and
+choose "Import with beets".
+
+Configuring
+-----------
+
+1. You'll want to set a few basic options before you start using beets. The
+:doc:`configuration </reference/config>` is stored in a text file. You
+can show its location by running ``beet config -p``, though it may not
+exist yet. 
+
+**Note:** You will need a text editor for these next steps. If you don't already have one, some popular ones are: `VS Code`_ , `Vim`_, and `Sublime`_. VS Code and Vim are free, but Sublime may come with a price tag. However, if you aren't too keen on downloading one, Windows Notepad will work in this case. 
```

Comment:
I don't think this is required. Windows comes with a number of text editors what will work just fine for these. We definitely don't need to be recommending a paid application.